Content.Server/Forensics/Components/FiberComponent.cs

```diff
@@ -0,0 +1,9 @@
+namespace Content.Server.Forensics
+{
+  [RegisterComponent]
+  public sealed class FiberComponent : Component
+  {
+    [DataField("fiberDescription")]
+    public string FiberDescription = "synthetic fibers";
+  }
+}
```

Comment:
indentation is incorrect in this file, same with `FingerprintComponent` and `FingerprintMaskComponent`